src/Adapter/Product/Combination/Repository/CombinationMultiShopRepository.php

```diff
@@ -426,7 +436,7 @@ public function getCombinationIds(ProductId $productId, ShopConstraint $shopCons
             ->andWhere('pas.id_product = :productId')
             ->andWhere($qb->expr()->in('pas.id_shop', ':shopIds'))
             ->setParameter('shopIds', $shopIds, Connection::PARAM_INT_ARRAY)
-            ->setParameter('productId', $productId->getValue())
+            ->setParameter('productId', $productId)
```

Comment:
```suggestion
            ->setParameter('productId', $productId->getValue())
```
I think we need the integer value here